docs/node.md

```diff
@@ -204,6 +204,12 @@ node {
   // If false, it will try to use globally installed node.
   download = true
 
+  //if false (the default), it will enable Gradle Task Rules for `yarn_*`, `npm_*` and `npm_*` task names,
+  //to support automatic task creation.
+  //if true, it will not register task rules when the plugin is applied, 
+  //so all rule-generated tasks will need to be configured manually (excluding static `yarn_setup`,`node_setup`,`npm_install`,`yarn_install`)   
```

Comment:
the static tasks don't use `snake_case`, they're all `camelCase`. e.g. `yarnInstall`, `npmInstall`, `nodeSetup`